internals/_init.php

```
@@ -0,0 +1,271 @@
+    // This is for in case of
+    // CBZ_DIR = /path/to/cbz/dir/
+    // $real_path = /path/to/cbz/dir
+    if (is_dir($real_path) && (!str_ends_with($real_path, '/') && !str_ends_with($real_path, '\\'))) {
+        $real_path .= DIRECTORY_SEPARATOR;
+    }
+    
+    if (str_starts_with($real_path, $cbz_dir)) {
+        return true;
+    } else {
+        return false;
+    }
```

PHPMD (2.15.0)

[error] The method is_safe_path uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)